src/compiler/index.ts

```diff
@@ -21,41 +20,35 @@ import { CompilerInfo } from './utils/types';
 azle();
 
 async function azle(): Promise<void> {
-    // We must run this before getNamesBeforeCli because
+    // We must run this before getNames because
```

Comment:
We can however move this into handleCli now

Reply:
Done